test/requests/search_tests.py

```diff
@@ -0,0 +1,45 @@
+import requests
+
+searches = [
+    { 
+        "found_text": "1 records were found",
+        "data": {
+            "species": "mouse",
+            "group": "BXD",
+            "type": "Hippocampus mRNA",
+            "dataset": "HC_M2_0606_P",
+            "search_terms_or": "PD-1",
+            "search_terms_and": ""
+        }
+    },
+    { 
+        "found_text": "4 records were found",
+        "data": {
+            "species": "mouse",
+            "group": "BXD",
+            "type": "Hippocampus mRNA",
+            "dataset": "HC_M2_0606_P",
+            "search_terms_or": "Ank-1",
+            "search_terms_and": ""
+        }
+    },
+    { 
+        "found_text": "1017 records were found",
+        "data": {
+            "species": "mouse",
+            "group": "BXD",
+            "type": "Hippocampus mRNA",
+            "dataset": "HC_M2_0606_P",
+            "search_terms_or": "sh*",
+            "search_terms_and": ""
+        }
+    }
+]
+
+def check_fulltext_searches(host):
+    for this_search in searches:

```

Comment:
**nitpick**: Try (same case for me) to be more mindful of naming. `this_search` can be off. How about:
```suggestion
    for search in searches:

```